src/App.js

```diff
@@ -13,7 +14,29 @@ function App() {
         <Route path="/metric" exact component={MetricPage} />
       </Switch>
     </div>
-  );
+  )
+
 }
 
+// function App() {
+//   return (
```

Comment:
you can just remove all of these comments (these are boilerplate codes, we don't need them)